### PR TITLE
consider this, or else...

### PR DESCRIPTION
**Pull request by markotarnaj**

### CODE REVIEW

File: index.scss [hunk 1]
```diff
@@ -1,127 +1,35 @@
-
-
 /**
```
Comment (flauc) [inline index.scss:1]: I think this would be better placed in an `.md` file.

File: scss/05-utilities/_utilities.scss [hunk 2]
```diff
@@ -59,109 +108,157 @@
...
-// Paddings
-.p-a-0 { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
+
+
+/* Paddings
+ *============================================================================*/
+
+.p-a-0   { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
+
 .p-a-xs  { @include fluid(10px, 5px, padding); }
```
Comment (flauc) [inline scss/05-utilities/_utilities.scss:162]: We have `xxl` but we don't have `xxs` what happens with people with OCD who see this? You ignorant bastard.

Comment (flauc) [reply scss/05-utilities/_utilities.scss:162]: Should we also have an option to add paddings/margins that aren't fluid?

File: scss/05-utilities/_utilities.scss [hunk 3]
```diff
@@ -59,109 +108,157 @@
...
-.show-m  { display: none; @media (max-width: 450px) { display: inline-block; } }
 
-// Paddings
-.p-a-0 { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
+
+
+/* Paddings
+ *============================================================================*/
+
+.p-a-0   { padding-top: 0; padding-right: 0; padding-bottom: 0; padding-left: 0; }
```
Comment (flauc) [inline scss/05-utilities/_utilities.scss:160]: why not `padding: 0`?

File: scss/05-utilities/_utilities.scss [hunk 4]
```diff
@@ -59,109 +108,157 @@
...
+.obj-contain { object-fit: contain; }
+
+
+
+
+
+/* Position, width, height
+ *============================================================================*/
+
+.relative { position: relative; }
```
Comment (flauc) [inline scss/05-utilities/_utilities.scss:128]: Why no `absolute` and `fixed` classes?

File: scss/05-utilities/_utilities.scss (hunk 3 above)
Comment (markotarnaj) [reply scss/05-utilities/_utilities.scss:160]: Because it doesn't overwrite "padding-side: value;"

File: scss/05-utilities/_utilities.scss (hunk 4 above)
Comment (markotarnaj) [reply scss/05-utilities/_utilities.scss:128]: Because they are usually followed by other attributes e.g. top, left, width.

**Pull request merged**